### DISCUSSION TITLE
Feature Request: Edit Embeds Already Posted

### DISCUSSION THREAD

**Discussion by KumariAshamaki** [Glitchii/embedbuilder#29] [Ideas]
Like the title says. I would love to replace CarlBot as all I use it for is embeds. CarlBot has commands to edit embeds it has posted such as.
embedsource <msg_ID> [channel]
then running ecembed <msg_ID> <channel> <JSON/Pastebin_Link/Starb.in_Link>
to edit the updated embed if this could be done in the editor even better but I would settle for commands :D

**Comment by Glitchii** [comment]
https://glitchii.github.io/embedbuilder is a website not a bot, so I don't think I can do the commands part, well, unless you're referring to my Discord bot or its website then.

I had already thought of this feature but It's quite impossible to add. For this feature to be possible, I would need to have a bot in your server which would do the work of getting the message from the channel and getting it's JSON content. Let me know if I'm wrong (or any work around) but it is not possible to read a message if you (or a bot) is not in the server. We can get emojis in any server however because Discord makes that possible through the API.

**Reply by KumariAshamaki** [reply]
Haha im sorry I was looking up a bot for this and found your site and thought it was a bot my bad.
I should have looked at the code XD
